src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService.cs

```diff
@@ -139,31 +144,36 @@ public class ThemeService : IThemeService
                         q => q.Include(t => t.Topics)
                     )
                 )
-                .OnSuccessVoid(
-                    async theme =>
-                    {
-                        // For now we only want to delete test themes as we
-                        // don't really have a mechanism to clean things up
-                        // properly across the entire application.
-                        // TODO: EES-1295 ability to completely delete releases
-                        if (!theme.Title.StartsWith("UI test theme"))
-                        {
-                            return;
-                        }
+                .OnSuccessDo(CheckCanDeleteTheme)
+                .OnSuccessVoid(async theme =>
```

Comment:
Just for my own curiosity, I think in other places we do stuff like this `CheckCanDeleteTheme` would return a theme and would be contained in a `OnSuccess`. How come the `OnSuccessDo` here?